docs/src/_docs/guide.md

```diff
@@ -140,7 +142,7 @@ Having multiple chains in the same object is valuable for evaluating convergence
 
 Please note that Turing does not have native support for chains sampled in parallel.
 
-### Sampling from the Prior
+### Sampling from the Unconditional Prior
```

Comment:
It sounds a bit strange to say "Unconditional Prior". Since in most cases, "Prior" refers to the distribution before conditioning on any data. Perhaps consider rewording the title to something else, e.g. "Sampling from the Unconditional Distribution (i.e. the prior)".